agave/tools/sync/sqs_client.py

```
@@ -0,0 +1,48 @@
+from dataclasses import dataclass, field
+from typing import Optional, Union
+from uuid import uuid4
+
+try:
+    import boto3
+    from types_boto3_sqs import SQSClient as Boto3SQSClient
+except ImportError:
+    raise ImportError(
+        "You must install agave with [sync_aws_tools] option.\n"
+        "You can install it with: pip install agave[sync_aws_tools]"
+    )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)